.gitignore

```diff
@@ -39,3 +39,5 @@ support/stream-circe/lib_managed/
 support/stream-circe/src_managed/
 support/stream-circe/project/boot/
 support/stream-circe/project/plugins/project/
+
+.idea/**
```

Comment:
Please use a global `.gitignore` (see https://stackoverflow.com/a/7335487/1519631) and remove this line.

Its not best practice to put IDE specific paths in `.gitignore`